### DISCUSSION TITLE
is a collection the same as a shopping list?

### DISCUSSION THREAD

**Discussion by ln206** [saleor/saleor#8131] [Q&A] [answered]
Also, is checkout the same as a shopping cart?

**Comment by karolkielecki** [comment, marked as answer]
Collections allow you to group your products, e.g., Summer Collection. You can find more info here: https://docs.saleor.io/docs/3.0/dashboard/catalog/collections

Checkout is a step-by-step process where a customer must follow to purchase the items, e.g., choosing a payment option, delivery option, etc.

**Comment by maarcingebala** [comment]
@ln206 Please check the Checkout docs for more details where it is also explained: https://docs.saleor.io/docs/3.0/developer/checkout. We don't distinguish between checkout and shipping cart; checkout serves the same purpose as cart in many shops.
